5.5/apache/Dockerfile

```diff
@@ -27,6 +27,15 @@ RUN a2enmod rewrite headers
 COPY magento.conf /etc/apache2/conf-enabled/
 
 
+ENV ION_CUBE_PHP_VERSION "5.5"
+ENV PHP_EXTENSION_DIR "/usr/local/lib/php/extensions/no-debug-non-zts-20121212"
```

Comment:
Looking at how the `docker-php-ext-enable` works, we could go about creating a bash script to do this for us, I'd be able to clean up what is in the dockerfile then too, wouldn't need any of the environment variables. (https://github.com/docker-library/php/blob/master/docker-php-ext-enable#L4)